### PR TITLE
Set timeout to a larger value

### PR DESCRIPTION
**Pull request by feiskyer**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

/kind failing-test

**What this PR does / why we need it**:

The current e2e tests are pretty flaky, set the timeouts to large values for reducing the failing tests.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```
NONE
```

### CODE REVIEW

File: tests/e2e/utils/service_utils.go [hunk 1]
```diff
@@ -28,6 +28,10 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 )
 
+const (
+	serviceTimeout = 20 * time.Minute
```
Comment (andyzhangx) [inline tests/e2e/utils/service_utils.go]: 10 min to delete a service is not enough?

Comment (feiskyer) [reply tests/e2e/utils/service_utils.go]: it should work for deletion, but for creation, 10 minutes may be not working

**Comment by feiskyer** [comment]
The autoscaling tests is still failing, but the basic test has been passed now.

@ritazh Let's get this in first? I think there may be some edge cases not handled correctly in autoscaling tests. We may fix that in another PR.
